CONTRIBUTING.md

```diff
@@ -17,6 +17,8 @@ Additional resources specific to the packages repository:
   As explained in the Flutter guide,
   [**PRs need tests**](https://github.com/flutter/flutter/wiki/Tree-hygiene#tests), so
   this is critical to read before submitting a plugin PR.
+- [Plugin tool](https://github.com/flutter/packages/tree/main/script/tool),
```

Comment:
This is covered in other links above; is there a specific reason to add it? I deliberately try to keep this list short to focus attention on the most important entry points.

Reply:
The current docs feel a bit fragmented. From skimming around I missed this tool existed. I'll revert this though.

Reply:
Reverted!